ddsrouter_core/test/blackbox/ddsrouter_core/dds/local/DDSTestLocalDisposeKey.cpp

```diff
@@ -19,17 +19,18 @@
 #include <gtest/gtest.h>
 
 #include <cpp_utils/testing/LogChecker.hpp>
-
-#include <ddsrouter_core/core/DDSRouter.hpp>
-#include <ddsrouter_core/types/topic/filter/WildcardDdsFilterTopic.hpp>
-#include <ddsrouter_core/types/topic/filter/DdsFilterTopic.hpp>
 #include <cpp_utils/Log.hpp>
 
+#include <ddspipe_core/types/topic/filter/WildcardDdsFilterTopic.hpp>
+
+#include <ddsrouter_core/core/DdsRouter.hpp>
+
 #include <test_participants.hpp>
 
-namespace eprosima {
-namespace ddsrouter {
-namespace core {
+using namespace eprosima;
+using namespace eprosima::ddspipe;
+using namespace eprosima::ddsrouter::core;
```

Comment:
I'm not very confident of these `using` as I'll prefer to implicit calls to `ddspipe::...` and `ddsrouter::....`. In case you want to keep them, check that they are repeated at the end of this file.

Reply:
It was the fastest way to refactor these files